### PR TITLE
Docs: Restructure governance section

### PR DESCRIPTION
**Pull request by dcwalk**
This PR focuses more on the page structure, without making many copy edits (I wanted to handle those separately if people would like to do line-level review). This PR:

- Cleans up and orders the pages in the menu
- Shortens many page names
- Aligns file naming with rest of docs (dashes vs. underscore)
- Moves best practices materials into one doc + updates references to new page
- Moves formatting materials into one doc + updates references to new page
- Removes the text proposal stub (nothing was written there)

New menu: 
Screenshot_2021-12-17_10-44-16

### CODE REVIEW

File: docs/governance/formatting.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+order: 4
+---
+
+# Formatting a Proposal
+
```
Comment (okwme) [inline docs/governance/formatting.md:6]:
```suggestion

Many proposals allow for long form text to be included, usually under the key `description`. These provide the opportunity to include [markdown](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) if formatted correctly as well as line breaks with `\n`. Beware, however, that if you are using the CLI to create a proposal, and setting `description` using a flag, the text will be [escaped](https://en.wikipedia.org/wiki/Escape_sequences_in_C) which may have undesired effects. If you're using markdown or line breaks it's recommended to put the proposal text into a json file and include that file as part of the CLI proposal, as opposed to individual fields in flags.

```

File: docs/governance/params-change/README.md [hunk 2]
```diff
@@ -52,20 +58,23 @@ The criteria for submitting a parameter-change proposal and the subsequent votin
 
 If a paramater-change proposal is successful, the change takes effect immediately upon completion of the voting period.
 
-## Note
+### Note
+
 - You cannot currently query the `bank` module's parameter, which is `sendenabled`. You also cannot query the `crisis` module's parameters.
-- You will need to compile [`gaiad`](./gaiad) from source into a binary file executable by your operating system eg. MacOS, Windows, Linux
+- You will need to compile [`gaiad`](../../getting-started/installation.md) from source into a binary file executable by your operating system eg. MacOS, Windows, Linux
 - You will need to indicate which chain you are querying, and currently this is `--chain-id cosmoshub-3`
```
Comment (okwme) [inline docs/governance/params-change/README.md]:
```suggestion
- You will need to indicate which chain you are querying, and currently this is `--chain-id cosmoshub-4`
```

File: docs/governance/params-change/README.md [hunk 3]
```diff
@@ -52,20 +58,23 @@ The criteria for submitting a parameter-change proposal and the subsequent votin
...
 - You will need to connect to a full node. If gaiad isn't already configured for this, you can use this tag in your command `--node [address]:26657`.
 
 ## Full nodes
+
 Running a full node can be difficult for those not technically-inclined, so you may choose to use a third-party's full node. In this case, the primary security risk is that of censorship: it's the single place where you have a single gateway to the network, and any messages submitted through an untrusted node could be censored.
 - http://51.79.82.228:26657 ([Figment Networks](https://figment.network/networks/cosmos/))
```
Comment (okwme) [inline docs/governance/params-change/README.md]:
```suggestion
- https://rpc.cosmos.network:443 ([Tendermint, Inc](https://tendermint.com))
```

File: docs/governance/params-change/README.md [hunk 4]
```diff
@@ -52,20 +58,23 @@ The criteria for submitting a parameter-change proposal and the subsequent votin
...
 
-## Contributors
-This documentation was created by Gavin Birch ([Figment Networks](https://figment.network)). Its development was supported by community funding that was approved on January 29, 2020 by the Cosmos Hub via community-spend proposal [Prop23](https://hubble.figment.network/cosmos/chains/cosmoshub-3/governance/proposals/23). You can read the [full proposal PDF here](https://ipfs.io/ipfs/QmSMGEoY2dfxADPfgoAsJxjjC6hwpSNx1dXAqePiCEMCbY). You can also create an issue or pull request to participate in its development at any time!
+## Credits
+
+This documentation was originally created by Gavin Birch ([Figment Networks](https://figment.network)). Its development was supported by funding approved on January 29, 2020 by the Cosmos Hub via Community Spend [Proposal 23](https://hubble.figment.network/cosmos/chains/cosmoshub-3/governance/proposals/23) ([full Proposal PDF here](https://ipfs.io/ipfs/QmSMGEoY2dfxADPfgoAsJxjjC6hwpSNx1dXAqePiCEMCbY)).
```
Comment (okwme) [inline docs/governance/params-change/README.md]:
```suggestion
This documentation was originally created by Gavin Birch ([Figment Networks](https://figment.network)). Its development was supported by funding approved on January 29, 2020 by the Cosmos Hub via Community Spend [Proposal 23](https://hubble.figment.network/cosmos/chains/cosmoshub-3/governance/proposals/23) ([full Proposal PDF here](https://ipfs.io/ipfs/QmSMGEoY2dfxADPfgoAsJxjjC6hwpSNx1dXAqePiCEMCbY)). In late 2021 and early 2022 significant updates were made by [Hypha, co-op](https://hypha.coop/), especially @dcwalk 🙏
```

**Comment by okwme** [comment]
looks great @dcwalk !
made a couple suggestions, what do you think?

**Comment by dcwalk** [comment]
Thanks @okwme ! Integrated your suggestions and used the updated language from main on the submitting page. I think this is good to merge once the checks pass :woman_dancing:

**Comment by dcwalk** [comment]
@uditvira maybe you are up for a review on this?

**Review by okwme — APPROVED** [review]

**Pull request merged**